CFC_WebApp/api/cfc_webapp.py

```diff
@@ -352,6 +354,10 @@ def javascriptCallback(clientName, method):
   client.callJavascriptCallback(client_key, method, request.params)
   return {'status': 'ok'}
 
+@get('/fbregister')
```

Comment:
My point is that I don't see anything in your code that makes a REST API call to:
https://<host.name>/fbregister
and until it does, there is no point in adding the API endpoint here.